.github/ISSUE_TEMPLATE/improvement.yaml

```diff
@@ -0,0 +1,21 @@
+name: Improvement
+title: "[Improvement] "
+description: Suggest an improvement on performance, code quality, user experience, etc.
+labels: [ "improvement" ]
+body:
+
+- type: textarea
+  attributes:
+    label: What would you like to be improved?
+    placeholder: >
+      Please describe the problem you see and how it is needed to be improved.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: How should we improve?
+    placeholder: >
+      Please provide the solution if you have thoughts.
```

Comment:
As above, if we are not sure how to improve the performance issue then I don't think we should be using issues